jiant/scripts/postproc/xtreme/README.md

```diff
@@ -0,0 +1,369 @@
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val,test \
+        --max_seq_length 256 \
+        --smart_truncate
+done
+```
+
+### UDPos
+```bash
+TASK=udpos_en
+python jiant/proj/main/tokenize_and_cache.py \
+    --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+    --model_type ${MODEL_TYPE} \
+    --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+    --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+    --phases train,val,test \
+    --max_seq_length 256 \
+    --smart_truncate
+for LG in af ar bg de el es et eu fa fi fr he hi hu id it ja ko mr nl pt ru ta te tr ur vi zh; do
+    TASK=udpos_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val,test \
+        --max_seq_length 256 \
+        --smart_truncate
+done
+for LG in kk th tl yo; do
+    TASK=udpos_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases test \
+        --max_seq_length 256 \
+        --smart_truncate
+done
+```
+
+### PANX
+```bash
+TASK=panx_en
+python jiant/proj/main/tokenize_and_cache.py \
+    --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+    --model_type ${MODEL_TYPE} \
+    --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+    --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+    --phases train,val,test \
+    --max_seq_length 256 \
+    --smart_truncate
+for LG in af ar bg bn de el es et eu fa fi fr he hi hu id it ja jv ka kk ko ml mr ms my nl pt ru sw ta te th tl tr ur vi yo zh; do
+    TASK=panx_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val,test \
+        --max_seq_length 256 \
+        --smart_truncate
+done
+```
+
+### XQuAD (uses SQuAD for training)
+```bash
+for LG in ar de el en es hi ru th tr vi zh; do
+    TASK=xquad_${LG}
+    python jiant/proj/main/tokenize_and_cache.py \
+        --task_config_path ${BASE_PATH}/tasks/configs/${TASK}_config.json \
+        --model_type ${MODEL_TYPE} \
+        --model_tokenizer_path ${BASE_PATH}/models/${MODEL_TYPE}/tokenizer \
+        --output_dir ${BASE_PATH}/cache/${MODEL_TYPE}/${TASK} \
+        --phases val \
+        --max_seq_length 384 \
```

Comment:
Is the different sequence length important?

Reply:
Yep. QA tasks generally use 384, while we usually default to 256 otherwise. Also, these hyperparams are taken from the reference implementation.